raiwidgets/raiwidgets/cohort.py

```diff
@@ -4,13 +4,20 @@
 """Module for defining cohorts in raiwidgets package."""
 
 import json
+import warnings
 from typing import Any, List, Optional
 
 import numpy as np
 import pandas as pd
 
 from responsibleai.exceptions import UserConfigValidationException
 
+warnings.warn(
```

Comment:
With the way python imports are structured, I'm a bit worried this will always be printed on importing raiwidgets.  Are you 100% sure this warning isn't always printed?  I don't think we want it to be printed on every import... only when methods/classes from cohort.py are actually used.

Reply:
yeah, import from that module only raises a warning.

(rai) C:\Users\gaugup\Documents\Github\responsible-ai-toolbox>python -Wd all
python: can't open file 'all': [Errno 2] No such file or directory
```
(rai) C:\Users\gaugup\Documents\Github\responsible-ai-toolbox>python -Wd
Python 3.7.11 (default, Jul 27 2021, 09:42:29) [MSC v.1916 64 bit (AMD64)] :: Anaconda, Inc. on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> from raiwidgets.cohort import Cohort, CohortFilter, CohortFilterMethods
C:\Users\gaugup\Anaconda3\envs\rai\lib\site-packages\flask\json\__init__.py:26: DeprecationWarning: Importing 'itsdangerous.json' is deprecated and will be removed in ItsDangerous 2.1. Use Python's 'json' module instead.
  _slash_escape = '\\/' not in _json.dumps('/')
C:\Users\gaugup\Anaconda3\envs\rai\lib\site-packages\flask\json\__init__.py:48: DeprecationWarning: Importing 'itsdangerous.json' is deprecated and will be removed in ItsDangerous 2.1. Use Python's 'json' module instead.
  class JSONEncoder(_json.JSONEncoder):
C:\Users\gaugup\Anaconda3\envs\rai\lib\site-packages\flask\json\__init__.py:84: DeprecationWarning: Importing 'itsdangerous.json' is deprecated and will be removed in ItsDangerous 2.1. Use Python's 'json' module instead.
  class JSONDecoder(_json.JSONDecoder):
MODULE-DEPRECATION-WARNING: The module raiwidgets.cohort is deprecated and will be removed in a later release. Please use raiutils.cohort instead.
>>> import raiwidgets
>>> from raiwidgets import ResponsibleAIDashboard
>>>
```